### PR TITLE
ENH: Versioneer

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

**Comment by jGaboardi** [comment]
The `gitcount` notebooks may also need to be updated for the versioneer switch. See pysal/spaghetti#654, etc.

File: pysal/explore/momepy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from momepy.distribution import *
+from momepy.diversity import *
+from momepy.elements import *
+from momepy.graph import *
+from momepy.intensity import *
+from momepy.shape import *
+from momepy.utils import *
+from momepy.weights import *
+from momepy.preprocessing import *
+from momepy.coins import *
```
Comment (martinfleis) [inline pysal/explore/momepy/__init__.py:12]: I think that this shouldn't be part of this PR.

File: requirements.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@ pointpats>=2.2.0
 segregation>=2.0.0
 spaghetti>=1.6.2
 mgwr>=2.1.2
+momepy>=0.5.0
```
Comment (martinfleis) [inline requirements.txt:10]:
```suggestion
```
As above.

Comment (sjsrey) [reply requirements.txt:10]: Merging #1230 will incorporate those changes (i.e., these commits will be integrated there).

**Review by martinfleis — APPROVED** [review]

File: pysal/explore/momepy/__init__.py (hunk 1 above)
Comment (martinfleis) [reply pysal/explore/momepy/__init__.py:12]: > Merging #1230 will incorporate those changes (i.e., these commits will be integrated there).

Not in this case as this is the original version, not the streamlined one.

**Comment by jGaboardi** [comment]
> The `gitcount` notebooks may also need to be updated for the versioneer switch. See [pysal/spaghetti#654](https://github.com/pysal/spaghetti/pull/654), etc.

Did we verify that the `gitcount` notebooks run with `versioneer` for Meta PySAL?

**Review by jGaboardi — APPROVED** [review]
Conditional approval based on [this gitcount issue](https://github.com/pysal/pysal/pull/1231#issuecomment-932974969).

**Comment by sjsrey** [comment]
> > The `gitcount` notebooks may also need to be updated for the versioneer switch. See [pysal/spaghetti#654](https://github.com/pysal/spaghetti/pull/654), etc.
> 
> Did we verify that the `gitcount` notebooks run with `versioneer` for Meta PySAL?

The meta release uses the latest release tag of each of the subppackages. So until we have more of the subpackages released that use versioneer, we can't really test this yet?

**Comment by jGaboardi** [comment]
> > > The `gitcount` notebooks may also need to be updated for the versioneer switch. See [pysal/spaghetti#654](https://github.com/pysal/spaghetti/pull/654), etc.
> > 
> > 
> > Did we verify that the `gitcount` notebooks run with `versioneer` for Meta PySAL?
> 
> The meta release uses the latest release tag of each of the subppackages. So until we have more of the subpackages released that use versioneer, we can't really test this yet?

We'll just merge this PR and keep the `gitcount` issue on the radar then. Sound good?
